petri/src/vm/hyperv/hyperv.psm1

```diff
@@ -401,3 +401,62 @@ function Restart-OpenHCL
 
     $result | Trace-CimMethodExecution -CimInstance $guestManagementService -MethodName "ReloadManagementVtl" -TimeoutSeconds $TimeoutHintSeconds
 }
+
+function Get-ManagementVtlSettings
+{
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $true)]
+        [System.Object]
+        $Vm,
+        
+        [Parameter(Mandatory = $true)]
+        [string] $Namespace
+    )
+    
+    $guestManagementService = Get-VmGuestManagementService
+    $vmid = $Vm.Id.toString()
+    
+    $result = $guestManagementService | Invoke-CimMethod -name "GetManagementVtlSettings" -Arguments @{
+        "VmId"      = $vmid
+        "Namespace" = $Namespace
+    }
+    
+    $result | Trace-CimMethodExecution -CimInstance $guestManagementService -MethodName "GetManagementVtlSettings"
+    
+    return @{
+        "Settings" = $result.Settings
+        "CurrentUpdateId" = $result.CurrentUpdateId
+    }
+}
+
+function Set-ManagementVtlSettings
+{
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $true)]
+        [System.Object]
+        $Vm,
+        
+        [Parameter(Mandatory = $true)]
+        [string] $Namespace,
+        
+        [Parameter(Mandatory = $true)]
+        [byte[]] $Settings,
+        
+        [Parameter(Mandatory = $true)]
+        [uint64] $UpdateId
+    )
+    
+    $guestManagementService = Get-VmGuestManagementService
+    $vmid = $Vm.Id.toString()
```

Comment:
other code in this file has all lowercase `tostring()`. Match the style of existing code in this file (here and always)